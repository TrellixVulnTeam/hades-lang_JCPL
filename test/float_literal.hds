
extern def print_f32(f32): Void = print_f32
extern def print_f64(f64): Void = print_f64

def main(): Void {

    print_f32(23.45)
    print_f64(34.24)
}
