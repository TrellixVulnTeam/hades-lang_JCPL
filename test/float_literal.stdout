23.45
34.24